openreview/journal/invitation.py

```diff
@@ -1285,34 +1532,42 @@ def set_reviewer_assignment(self, assignment_delay):
             maxReplies=1,            
             type='Edge',
             edit={
+                'id': {
+                    'param': {
+                        'withInvitation': self.journal.get_reviewer_custom_max_papers_id(),
+                        'optional': True
+                    }
+                },                 
                 'ddate': {
-                    'type': 'date',
-                    'range': [ 0, 9999999999999 ],
-                    'optional': True,
-                    'nullable': True
-                },
-                'readers': {
-                    'const': [venue_id, action_editors_id, '${tail}']
-                },
-                'writers': {
-                    'const': [venue_id, '${tail}']
+                    'param': {
+                        'range': [ 0, 9999999999999 ],
+                        'optional': True,
+                        'deletable': True
+                    }
                 },
+                'readers': [venue_id, self.journal.get_action_editors_id(number='${{2/head}/number}'), '${2/tail}'],
```

Comment:
```suggestion
                'readers': [venue_id, self.journal.get_action_editors_id(), '${2/tail}'],
```

Reply:
This is giving me an error when trying to update a custom max papers edge since the head is a group and not a note

Reply:
thanks for the catch!